atlas/packages/app/src/utils/number.ts

```diff
@@ -14,5 +14,10 @@ export const formatNumberShort = (num: number): string => {
     suffix = 'K'
   }
 
-  return `${value % 1 === 0 ? value : value.toFixed(1)}${suffix}`
+  let formattedValue = value.toFixed(1)
+  if (formattedValue.endsWith('.0')) {
+    formattedValue = formattedValue.slice(0, formattedValue.length - 2)
```

Comment:
Formatting is still a bit inconsistent,  `1000` and `1001` will get two different representations, `1000` and `1K` respectively, same happens for `1000000` and `1000001`


```javascript
// This evaluates to "1000" 
formatNumber(1000)
// This evaluates to "1K"
formatNumber(1001)
```

Reply:
ofc, that should have been `>=` instead of `>`. Fixed now!